website/docs/reference/warehouse-profiles/bigquery-profile.md

```diff
@@ -6,16 +6,16 @@ title: "BigQuery Profile"
 
 BigQuery targets can be specified using one of three methods:
 
-1. oauth
+1. oauth via `gcloud`
+2. oauth via access token(s)
```

Comment:
The list here is `1, 2, 2, 3`. This ends up rendering correctly, but would be good to fix :)